openstudiocore/src/energyplus/ForwardTranslator.cpp

```diff
@@ -2463,6 +2463,12 @@ boost::optional<IdfObject> ForwardTranslator::translateAndMapModelObject(ModelOb
       retVal = translateScheduleFixedInterval(schedule);
       break;
     }
+  case  openstudio::IddObjectType::OS_Schedule_File:
+   {
+    model::ScheduleFile schedule = modelObject.cast<ScheduleFile>();
+    retVal = translateScheduleFile(schedule);
+    break;
+   }
   case  openstudio::IddObjectType::OS_Schedule_Rule :
```

Comment:
Add a no-op translation for ExternalFile to avoid warnings, in future an implementation could do file copy magic